app/src/main/res/layout/preference_cat_style.xml

```diff
@@ -7,7 +7,8 @@
     style="?android:attr/listViewStyle"
     android:layout_marginTop="16dp"
     android:textAlignment="viewStart"
-    android:layout_marginStart="16dp"
+    android:layout_marginStart="72dp"
```

Comment:
This cat_style used in this layouts but no worry because this change will align all preference category titles.
Maybe we can remove margin_bottom.

![resim](https://user-images.githubusercontent.com/17825767/85882957-db6ef880-b7e8-11ea-8c08-1b5e02a63b10.png)

Reply:
![Annotation 2020-06-26 201334](https://user-images.githubusercontent.com/17825767/85883365-88497580-b7e9-11ea-984f-83f96eddf10c.png)
![Annotation 2020-06-26 201420](https://user-images.githubusercontent.com/17825767/85883430-a57e4400-b7e9-11ea-8491-01435c327cc9.png)

Yes, we need to remove margin_bottom.